### PR TITLE
[docs] Update site content

### PR DESCRIPTION
**Pull request by z9r5**
Site content updates.

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: docs/site/_data/revision_comparison.yml
Comment (EvgenySamoylov) [inline docs/site/_data/revision_comparison.yml:1]: I think we need to add the link to the documentation section for each feature mentioned.

File: docs/site/_data/revision_comparison.yml [hunk 1]
```diff
@@ -137,6 +143,12 @@ sections:
         revision:
           ce: true
           ee: true
+      - description:
+          en: Out-of-the-box application delivery to the cluster
+          ru: Решение вопросов по доставке приложений в кластер из коробки
```
Comment (z9r5) [inline docs/site/_data/revision_comparison.yml:148]:
```suggestion
          en: Out-of-the-box application delivery to the cluster
          ru: Решение вопросов по доставке приложений в кластер из коробки
       url: /documentation/v1/modules/502-delivery/
```

File: docs/site/_data/revision_comparison.yml [hunk 2]
```diff
@@ -137,6 +143,13 @@ sections:
...
+       url: /documentation/v1/modules/502-delivery/
+        revision:
+          ce: false
+          ee: true
```
Comment (z9r5) [inline docs/site/_data/revision_comparison.yml:152]:
```suggestion
        url: /documentation/v1/modules/502-delivery/
        revision:
          ce: false
          ee: true
```

File: docs/site/_data/revision_comparison.yml
Comment (z9r5) [reply docs/site/_data/revision_comparison.yml:1]: Fixed

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Pull request merged**